contracts/fuelchain/FuelMessagePortal.sol

```diff
@@ -202,7 +188,7 @@ contract FuelMessagePortal is
                 abi.encodePacked(blockHeader.computeConsensusHeaderHash()),
                 blockInHistoryProof.proof,
                 blockInHistoryProof.key,
-                rootBlockHeader.height - 1
+                rootBlockHeader.height
```

Comment:
This was a bug that needed fixing so the tests would work. Our current integration tests relay directly from a block so this never popped up when running against an actual fuel chain that starts at block height 0 instead of 1 like was incorrectly being constructed for testing before.